### ISSUE TITLE
Support for fractional scaling settings

### ISSUE THREAD

**Issue by tiagozc** [jenskutilek/sudo-font#29]
Hi there. First, thanks for providing this font free of charge, it's great.

I'm reaching out directly to you because I'm running out of ideas to have your font working properly in my current setup. I have a 24" monitor, 2560x1440, which for me works best when the whole system is scaled to 125% (1.25). 

On my setup (scaled to 125%), when I use any electron app with Sudo font, the Sudo font does not scale and is too small when compared to others (it seems its kept at 100% scale). **That does not happen if I use a native app though. With native apps it scales properly.** So, I understand that this can be an issue with electron based apps, but I have tried multiple different settings on it, and what intrigues me most is that other fonts don't have the same problem as Sudo font.

For example, here's a comparison side by side of JetBrains Mono (left) and Sudo font (right), both 16px size on Visual Studio Code.

![out](https://github.com/user-attachments/assets/7931227f-d8ba-4a5e-8107-99f542595156)

That scaling problem happens on both Linux and Windows. I don't think it's an issue with your font, but at the same time I don't understand why it does not happen to others, like JetBrains Mono. Is there anything font wise that can be done to fix that?

Thank you.

**Comment by jenskutilek** [comment]
Sorry, I can't reproduce it.

It is true that Sudo will appear smaller than some other fonts, including JetBrains Mono, at the same pixel size. Maybe this is what you're seeing?

**Comment by chmielcode** [comment]
> Sorry, I can't reproduce it.
> 
> It is true that Sudo will appear smaller than some other fonts, including JetBrains Mono, at the same pixel size. Maybe this is what you're seeing?

This is true and it's one of the few fonts that have that issue. For example JetBrains Mono looks the same across apps with Notepad++ font size 10 and font size 13 everywhere (except some old apps) else. It's either 10 or 13 for the whole application and fonts used in other contexts size proportionately, so everything looks good. This is a problem since there is no way to e.g. force font size in Firefox for monospaced option only. 

For Sudo the "correct-for-me" font size in Notepad++ is 12 and in VSCode I get the same result with 16. I could not find a font to pair Sudo with in Firefox that is small enough so that bumping minimal font size wouldn't make sans and sans-serif options look gigantic. Another example is that pandas dataframe preview in VSCode uses other font and editor's font size, so it's huge in comparison to code.

**Comment by jenskutilek** [comment]
@chmielcode The size discrepancy has occasionally bothered me before ;) I've done some tests and will change the apparent size by adjusting the units per em value. Tracking the change in #37.

**Comment by jenskutilek** [comment]
@chmielcode The current font files in the repo have a different scaling setting so the apparent font size is better comparable with other common fonts. The next release will also have the new font size.
